packages/js-drive/lib/abci/handlers/deliverTxHandlerFactory.js

```diff
@@ -144,27 +144,28 @@ function deliverTxHandlerFactory(
     // in order to store them in credits distribution pool
     const actualStateTransitionFee = stateTransition.calculateFee();
 
-    if (actualStateTransitionFee > predictedStateTransitionFee) {
-      throw new PredictedFeeLowerThanActualError(
-        predictedStateTransitionFee,
-        actualStateTransitionFee,
-        stateTransition,
-      );
-    }
+    // TODO: enable once fee calculation is done
+    // if (actualStateTransitionFee > predictedStateTransitionFee) {
+    //   throw new PredictedFeeLowerThanActualError(
+    //     predictedStateTransitionFee,
+    //     actualStateTransitionFee,
+    //     stateTransition,
+    //   );
+    // }
 
-    const identity = await transactionalDpp.getStateRepository().fetchIdentity(
-      stateTransition.getOwnerId(),
-    );
+    // const identity = await transactionalDpp.getStateRepository().fetchIdentity(
+    //   stateTransition.getOwnerId(),
+    // );
 
-    const updatedBalance = identity.reduceBalance(actualStateTransitionFee);
+    // const updatedBalance = identity.reduceBalance(actualStateTransitionFee);
 
-    if (updatedBalance < 0) {
-      throw new NegativeBalanceError(identity);
-    }
+    // if (updatedBalance < 0) {
+    //   throw new NegativeBalanceError(identity);
+    // }
 
-    await transactionalDpp.getStateRepository().updateIdentity(identity);
+    // await transactionalDpp.getStateRepository().updateIdentity(identity);
 
-    blockExecutionContext.incrementCumulativeFees(actualStateTransitionFee);
+    blockExecutionContext.incrementCumulativeProcessingFee(actualStateTransitionFee);
```

Comment:
If you look bellow you will find:

```js
    const {
      storageFee: actualStorageFee,
      processingFee: actualProcessingFee,
    } = calculateOperationFees(actualStateTransitionOperations);
```

That's what you need to increment both processing and storage fees